### PR TITLE
Added Options for additional lavalink v2

### PR DESCRIPTION
**Pull request by DarrenOfficial**
added support for backup / secondary lavalink options, if one is down it will go through the next lavalink in the config.js

Please move lines that apply to you out of the comment:

- Code changes have been tested against the Discord API
- Code changes has been tested for every lavalink schenario that I can think of
- I know how to update typings and have done so, or typings don't need updating

### CODE REVIEW

**Comment by DarrenOfficial** [comment]
Do not merge this with the main brance, i just found out that spotify playback is not working, only youtube works.

File: commands/play.js [hunk 1]
```diff
@@ -45,6 +42,17 @@ module.exports = {
             if (SearchString.match(client.Lavasfy.spotifyPattern)) {
                 await client.Lavasfy.requestToken();
                 let node = client.Lavasfy.nodes.get(client.config.Lavalink.id);
+                const temp = 1;
+                var trueass = new Boolean(False);
```
Comment (SudhanPlayz) [inline commands/play.js:46]:
```suggestion
                var trueass = false;
```

File: commands/play.js [hunk 2]
```diff
@@ -143,8 +147,17 @@ module.exports = {
             if (search.match(client.Lavasfy.spotifyPattern)) {
                 await client.Lavasfy.requestToken();
                 let node = client.Lavasfy.nodes.get(client.config.Lavalink.id);
-                let Searched = await node.load(search);
-
+                const temp = 1;
+                var trueass = new Boolean(False);
```
Comment (SudhanPlayz) [inline commands/play.js:151]:
```suggestion
                var trueass =false
```

File: config.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 module.exports = {
   Admins: ["UserID", "UserID"], //Admins of the bot
-  ExpressServer: true,//If you wanted to make the website run or not
+  ExpressServer: false,//If you wanted to make the website run or not
```
Comment (SudhanPlayz) [inline config.js:3]: Hm why?

File: config.js [hunk 4]
```diff
@@ -18,14 +18,23 @@ module.exports = {
 
   //Lavalink
    Lavalink: {
```
Comment (SudhanPlayz) [inline config.js:20]: Array is always better than a group of objects

**Review by SudhanPlayz — CHANGES_REQUESTED** [review]
I am not accepting this due to using objects instead of arrays

**Pull request closed without merging**